src/app/feature/calendar/cards/christmas-tree/christmas-tree.component.html

```diff
@@ -1149,42 +1146,15 @@
         ></div>
       </div>
       <div class="tree-top">
-        <div
-          class="star"
-          style="transform: translateZ(225px) translateY(-4.25px) rotateX(-90deg)"
-        ></div>
-        <div
-          class="star"
-          style="transform: translateZ(225px) translateY(-4.5px) rotateX(-90deg)"
-        ></div>
-        <div
-          class="star"
-          style="transform: translateZ(225px) translateY(-4.75px) rotateX(-90deg)"
-        ></div>
-        <div
-          class="star"
-          style="transform: translateZ(225px) translateY(-5px) rotateX(-90deg)"
-        ></div>
-        <div
-          class="star"
-          style="transform: translateZ(225px) translateY(-5.25px) rotateX(-90deg)"
-        ></div>
-        <div
-          class="star"
-          style="transform: translateZ(225px) translateY(-5.5px) rotateX(-90deg)"
-        ></div>
-        <div
-          class="star"
-          style="transform: translateZ(225px) translateY(-5.75px) rotateX(-90deg)"
-        ></div>
-        <div
-          class="star"
-          style="transform: translateZ(225px) translateY(-6px) rotateX(-90deg)"
-        ></div>
-        <div
-          class="star"
-          style="transform: translateZ(225px) translateY(-6.25px) rotateX(-90deg)"
-        ></div>
+        <div class="star" style="transform: translateZ(225px) translateY(-4.25px) rotateX(-90deg)"></div>
+        <div class="star" style="transform: translateZ(225px) translateY(-4.5px) rotateX(-90deg)"></div>
+        <div class="star" style="transform: translateZ(225px) translateY(-4.75px) rotateX(-90deg)"></div>
+        <div class="star" style="transform: translateZ(225px) translateY(-5px) rotateX(-90deg)"></div>
+        <div class="star" style="transform: translateZ(225px) translateY(-5.25px) rotateX(-90deg)"></div>
+        <div class="star" style="transform: translateZ(225px) translateY(-5.5px) rotateX(-90deg)"></div>
+        <div class="star" style="transform: translateZ(225px) translateY(-5.75px) rotateX(-90deg)"></div>
+        <div class="star" style="transform: translateZ(225px) translateY(-6px) rotateX(-90deg)"></div>
+        <div class="star" style="transform: translateZ(225px) translateY(-6.25px) rotateX(-90deg)"></div>
```

Comment:
`@for`? 

I see these stars have incremental `translateY` property. You can do it way better.

Reply:
Same would apply for the entire component I think, such refactor would be better in another PR I guess as this one is mostly about migrations ^^